virt/getting_started/virt-web-console-overview.adoc

```diff
@@ -252,6 +252,11 @@ Preview features are disabled by default and must not be enabled in production e
 |*Live Migration* -> *Live migration network* list
 |Select a dedicated secondary network for live migration.
 
+|*SSH Configuration* -> *SSH over LoadBalancer service* toggle
```

Comment:
Responding to request for review. The "toggle" is a switch. See the [UI guidelines](https://docs.engineering.redhat.com/pages/viewpage.action?spaceKey=CCSENABLE&title=UI+guidelines%3A+Quick+reference), which links to the IBM Style Guide if you want to see the original guideline.

Normally, we don't mention an interface element, but in this case it does make sense to say `*SSH over LoadBalancer service* switch`.